index.js

```diff
@@ -75,7 +76,11 @@ class ServerlessPlugin {
           if (this.logging) {
             this.serverless.cli.log('\t - ' + key)
           }
-          this.serverless.service.provider.environment[key] = envVars[key]
+          if (processEnvOnly) {
+            process.env[key] = envVars[key]
```

Comment:
@miguel-a-calles-mba I just remembered that when executing the dotenv function, it already adds vars to process.env. There's no need to manually do it. Can you just change this so that it's

```
if (!processEnvOnly) {
   this.serverless.service.provider.environment[key] = envVars[key]
}
```

Reply:
Also, make sure you pull fresh from master. There's a conflict now but doesn't look like a problem to clean up.